App/containers/AllEventsScreen/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React, {useState, useEffect} from 'react';
+import {FlatList, StyleSheet, Text, View} from 'react-native';
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  Header,
+  Icon,
+  Left,
+  List,
+  ListItem,
+  Right,
+  Title,
+} from 'native-base';
+import {DrawerActions} from 'react-navigation-drawer';
+import api from '../../utils/api';
+import {NavigationStackProp} from 'react-navigation-stack';
+import dayjs from 'dayjs';
+import {ApiResponse} from 'apisauce';
+
+type Props = {
+  navigation: NavigationStackProp;
+};
+const AllEvents = ({navigation}: Props) => {
+  const [events, setMyEvents] = useState([]);
```

Comment:
Mozemy dodac typowanie w ten sposob
```suggestion
  const [events, setMyEvents] = useState<EventType[]>([]);
```